07-final-project/components/character-info-list.js

```diff
@@ -0,0 +1,29 @@
+var React = require('react');
+
+var CharacterInfoList = React.createClass({
+
+    render: function () {
+
+        return (
+            <div>
+                    {this.renderCharacterInfo()}
+            </div>
+        );
+    },
+
+    renderCharacterInfo: function () {
+        return (
+            <ul>
+
+                <li>Name: {this.props.data.name}</li>
+                <li>Realm: {this.props.data.realm}</li>
```

Comment:
why use data instead of setting multiple props? Maybe it's OK but I want to know why